src/static/css/index.css

```diff
@@ -1,20 +1,26 @@
-header {
-  /* background-image: url(/static/images/intro-background-fit.svg); */
-  background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxOTIwIiBoZWlnaHQ9IjEyNTAiIHZpZXdCb3g9IjAgMCAxOTIwIDEyNTAiIHByZXNlcnZlQXNwZWN0UmF0aW89Im5vbmUiPjxkZWZzPjxzdHlsZT4uY2xzLTF7ZmlsbDojYThjYWJhO2ZpbGwtcnVsZTpldmVub2RkO29wYWNpdHk6LjA1fTwvc3R5bGU+PC9kZWZzPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS0yMzEgODloMjA4OHY2NTFsLTIwODggMzYwVjg5eiIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS00NzAgMjM5aDIwODh2NjUxbC0yMDg4IDM2MFYyMzl6Ii8+PHBhdGggZD0iTS0xNDQtMTloMjA4OHY2NTFMLTE0NCA5OTJWLTE5eiIgZmlsbD0iIzFhMmI0OSIgb3BhY2l0eT0iLjciIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjxwYXRoIGNsYXNzPSJjbHMtMSIgZD0iTS00NzAtODNoMjA4OHY2NTFMLTQ3MCA5MjhWLTgzeiIvPjxwYXRoIGQ9Ik0tMjMyLTQ4aDIwODh2NjUxTC0yMzIgOTYzVi00OHoiIGZpbGw9IiNmMmYyZjIiIG9wYWNpdHk9Ii4wNSIgZmlsbC1ydWxlPSJldmVub2RkIi8+PHBhdGggZD0iTS0xNDI2LTM3MUg2NjJ2NjUxbC0yMDg4IDM2MFYtMzcxeiIgb3BhY2l0eT0iLjEiIGZpbGw9IiNmMmYyZjIiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==);
+header.alt-bg {
```

Comment:
Usually, when adding background fallbacks you add `background-color: ...`  to render a bg color until the HTTP request is made and proper bg image loads, next a  `background: ...` shorthand or the safer `background-image`.

I am not sure I understand the fallback order in this CSS rule but I am sure I might missing out bits here. The way I see and understand these lines, please correct me if I'm wrong:

`background-color: #5C687D;`
The classic bg color fallback.

`background: transparent linear-gradient(#5C687D 70%, transparent 30%);` 
This shorthand overrides the above completely and usually, the gradient is seen as a progressive enhancement when the browser has better support than the simple color background. I feel that's not really the case here, as we're not supporting old browsers at all, at this time (e.g. by using CSS grid and flexbox for the site's structure without any fallbacks).

`background: transparent url(/static/images/intro-background-fit.svg) bottom left / 100% 100% no-repeat;`
Isn't this completely overridden by the next background Base64 declaration? Also, I didn't checkout this branch yet, but, if I'm not mistaken, this allows an extra  `/static/images/intro-background-fit.svg`, HTTP request which we've been getting rid of it previously.

Reply:
So my thinking here is:
`background-color: #5C687D;`
Fall back when all else fails. However it fills up too much of the screen and bleeds into the Featured Chapter section so not ideal. Which leads on to:
`background: transparent linear-gradient(#5C687D 70%, transparent 30%);`
A slightly nicer fall back which only covers 70% of the screen when `linear-gradient` is supported. It's not actually a gradient as it has hard cut offs.
Then the SVGs.

> Isn't this completely overridden by the next background Base64 declaration?

Yes. The URL is commented out so shouldnt cause a download. This is also how it exists as present so think you added it? I'm seeing it as a bit of hint/comment as to where that base64 is derviced from so if anyone (like me in this PR!) wants to change it, then they know were to get the source from.

Reply:
> I feel that's not really the case here, as we're not supporting old browsers at all, at this time (e.g. by using CSS grid and flexbox for the site's structure without any fallbacks).

I think this is a big part of what I'm trying to solve here. It's find not supporting grid if the cotent is still readable. For example, IE11 disables the home page, but without laying it out properly and without the fonts but it's still readable. That's fine and the page is progressively enhanced for other browsers. We don't not support IE11 - we just have a better experience when using a better browser.

However if the SVG does not load (because AT doesn't support it, or you're in reader mode, or just due to a bug in a browsers SVG rendering engine) then the text is white on white and completely unreadable. Yes it may be unlikely but I don't know all the AT technologies out there, or any future bugs, and the fallbacks work, and are not hacky so why not have them?

So we should have progressive enhancement, but not cut off older browsers or other non-browsers if they don't support the code we use. That's the beauty of the web which degrades gracefully and this PR allows us to do this.

Reply:
Ah I see I forgot to comment this out again. Fixed now!